src/commands/edge_app_utils.rs

```diff
@@ -191,17 +191,19 @@ mod tests {
             entrypoint: Some("entrypoint.html".to_owned()),
             settings: vec![
                 Setting {
+                    name: "display_time".to_string(),
                     type_: SettingType::String,
                     default_value: Some("5".to_string()),
-                    title: "display_time".to_string(),
+                    title: "display time title".to_string(),
                     optional: true,
                     is_global: false,
                     help_text: "For how long to display the map overlay every time the rover has moved to a new position.".to_string(),
                 },
                 Setting {
+                    name: "google_maps_api_key".to_string(),
```

Comment:
this is getting confusing though with name and title. I wish we just had a field named `id` or something. Or "key" since it's a setting. key=val. Oh well. We could rename it here and pass the correct data to API but not sure if it's worth it

Reply:
Yeah - thought about that, but decided we could remember the name and title difference.
Also in the cli there is no name in the yaml anyway - I skip serialize and deserialize for the name